AppCenter/AppCenter/Internals/Sender/MSOneCollectorIngestion.h

```diff
@@ -0,0 +1,22 @@
+#import "MSHttpSender.h"
+
+static NSString *const kMSApiVersion = @"1.0";
+static NSString *const kMSApiPath = @"/OneCollector";
+static NSString *const kMSOneCollectorContentType = @"application/x-json-stream; charset=utf-8;";
+static NSString *const kMSApiKey = @"apikey";
+static NSString *const kMSClientVersionKey = @"Client-Version";
+static NSString *const kMSClientVersionFormat = @"ACS-iOS-ObjectiveC-no-%@-no";
```

Comment:
Can we add a comment explaining this format?
e.g. ```<sdktype>-<platform>-<language>-<projection>-<version>-<tag>```

Reply:
I found it [here](https://microsoft.sharepoint.com/:w:/r/teams/OneCollector/_layouts/15/WopiFrame.aspx?sourcedoc=%7Be335ec26-d207-4769-981e-eaf807cae7da%7D&action=edit&wdPid=33faa0d8). Section 3.4 Headers.  It is the same value as sdk.libVer.